src/frontend/Komponenter/Behandling/Simulering/SimuleringTabellWrapper.tsx

```diff
@@ -71,6 +76,13 @@ const SimuleringTabellWrapper: React.FC<{
     return (
         <SimuleringsContainer>
             <SimuleringOversikt simulering={simuleringsresultat} />
+            {simuleringsresultat.sumManuellePosteringer != null &&
```

Comment:
TS/JS har trippel-likhet; `!==`

Reply:
Eksplisitt sjekke for null og undefined ?

Reply:
Må kanskje det... Trekke det ut som en funksjon/konstant så blir det litt enklere å lese?